src/Controllers/ApiController.php

```diff
@@ -134,6 +135,10 @@ private function generateResponse(Request $request)
         }
 
         try {
+            $botLoginPromise = $this->client->tryBotLogin($this->session);
```

Comment:
No, we cant do this on every request.

We need to create new endpoint in router. For example /combined_api.

And in callApi method check which endpoint was requested. 
If combined_api endpoint requested, then instead of getInstance method, use directly MadelineProtoCombined property.